master/internal/experiment/bulk_action.go

```diff
@@ -246,6 +254,11 @@ func ActivateExperiments(ctx context.Context, system *actor.System,
 func CancelExperiments(ctx context.Context, system *actor.System,
 	experimentIds []int32, filters *apiv1.BulkExperimentFilters,
 ) ([]ExperimentActionResult, error) {
+	if filters != nil {
+		if filters.States == nil {
+			filters.States = []experimentv1.State{experimentv1.State_STATE_ACTIVE}
```

Comment:
question: currently on latest master it seems like we're able to cancel and kill paused experiments, but this looks to me like it's limiting us to only active experiments. I'm not the best at reading go though, so I could be wrong.

Reply:
oh this is a good point, I was thinking of filtering out the completed experiments and had not considered paused

Reply:
ok this issue is fixed on cancel and kill, will update PR message